### ISSUE TITLE
scientific notation in Correlogram

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#183]
```
Correlogram: distance values on horizontal axis of correlogram use scientific 
notation for values > 10,000, e.g., 1.131e+04. This takes up 8 characters, so 
scientific notation should only be implemented for values > 10,000,000 - may 
need to check the general formatting assumptions

```

Original issue reported on code.google.com by `lixun...@gmail.com` on 30 Jun 2015 at 7:27

**Comment by jkoschinsky** [comment]
Let's truncate labels on horizontal axis at 1 decimal place since they currently tend to overlap.

Which dataset are you using to test the 10k threshold?

I just found a replicable crash with the correlogram with Arc Distance (mi). Will email the data and specification separately since it's on non-public data.

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.87 (OSX Yosemite, 10.10.5).
